build.gradle

```diff
@@ -99,8 +99,8 @@ subprojects { subproject ->
 		log4j2Version = '2.7'
 		logbackVersion = '1.1.7'
 		mockitoVersion = '1.10.19'
-		rabbitmqVersion = project.hasProperty('rabbitmqVersion') ? project.rabbitmqVersion : '3.6.5'
-		rabbitmqHttpClientVersion = '1.0.0.RELEASE'
+		rabbitmqVersion = project.hasProperty('rabbitmqVersion') ? project.rabbitmqVersion : '4.0.0'
```

Comment:
I don't see any reason to - I guess we'll have to talk to @wilkinsona about the amqp-client version for boot 1.5, though.